File: src/main/java/com/monsource/bis/report/model/City.java
package com.monsource.bis.report.model;

import java.util.List;

public class City {

	private Integer id;
    private String name;
    private List<District> districts;

    public City() {
    }

    public City(Integer id, String name) {
        this.id = id;
        this.name = name;
    }

    public Integer getId() {
		return this.id;
	}

	public void setId(Integer id) {
		this.id = id;
	}

	public String getName() {
		return this.name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public List<District> getDistricts() {
		return this.districts;
	}

	public void setDistricts(List<District> districts) {
		this.districts = districts;
	}

}

File: src/main/webapp/app/report/model/City.js
Ext.define('Report.model.City', {
    extend: 'Ext.data.Model',
    fields: [
        {name: 'id', type: 'int'},
        {name: 'name', type: 'string'}
    ],
    hasMany: {model: 'Report.model.District', name: 'districts'}
})